hive/src/main/java/org/apache/iceberg/hive/CustomHiveCatalog.java

```diff
@@ -0,0 +1,30 @@
+package org.apache.iceberg.hive;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.iceberg.TableOperations;
+import org.apache.iceberg.catalog.TableIdentifier;
+
+
+/**
+ * A {@link HiveCatalog} which uses {@link CustomHiveTableOperations} underneath.
+ *
+ * This catalog is especially useful if the Iceberg operations are being performed in response to Hive operations
+ */
+public class CustomHiveCatalog extends HiveCatalog {
```

Comment:
Do you want to add an entry to `org.apache.iceberg.hive.HiveCatalogs` for caching custom catalogs?

Reply:
Done